app/Models/Tour.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Tour extends Model
+{
+    use HasFactory;
+
+    protected $fillable = [
+        'name',
+        'details',
+        'guest_house_id',
+        'location_id',
+        'sport_activity_id',
+    ];
+
+    public function guesthouses()
+    {
+        return $this->hasMany('GuestHouse');
```

Comment:
try to find where should be placed inversed relation and where direct
https://laravel.com/docs/8.x/eloquent-relationships#one-to-many

Reply:
Fixed the method name and relation